lib/gooddata.rb

```diff
@@ -8,8 +8,8 @@
 $pmap_default_thread_count = 20 # rubocop:disable GlobalVars
 
 # GoodData Module
-module GoodData
-end
+ module GoodData
+end 
```

Comment:
#<struct Pronto::Git::Line line=#<Rugged::Diff::Line:47112739888760 {line_origin: :addition, content: "end \n">, patch=#<struct Pronto::Git::Patch patch=#<Rugged::Patch:47112719978480>, repo=#<Pronto::Git::Repository:0x000055b291e0a440 @repo=#<Rugged::Repository:47112719979020 {path: "/src/.git/"}>>>, hunk=#<Rugged::Diff::Hunk:47112739962920 {header: "@@ -8,8 +8,8 @@ require 'pmap'\n", count: 10}>> warning: Lint/EndAlignment: `end` at 12, 0 is not aligned with `module` at 11, 1. (rubocop)